### ISSUE TITLE
Baseline training

### ISSUE THREAD

**Issue by ming666-wum** [imirzadeh/Teacher-Assistant-Knowledge-Distillation#15]
Hello, thank you for sharing good codes.
I've just tried to run baseline training code (Example 3.1), but right after running its status changes to this:

TypeError: unsupported operand type(s) for /:'Tensor' and 'NoneType'
![image](https://user-images.githubusercontent.com/59129055/80305341-c973d800-87ee-11ea-8086-c23627397280.png)

I attached images url.
I cannot figure out what the problem is... Do you know something about this issue?
Thank you very much.

**Comment by imirzadeh** [comment]
I'm not sure but I think the value 'T' here is None. what is the value of your config object?

**Comment by ming666-wum** [comment]
Thank you for your reply!
 I ran into this problem when I was reproducing your code. T is the T_student in the search_space.json , which is called in the train.py file. After searching and experimenting, I thought that the search_ space.json file was not read in by the train.py file, and the attempt to modify it failed. Do you have any suggestions or measures for this? I am looking forward to your reply.

**Comment by imirzadeh** [comment]
Yes, one fix could be to set the config for yourself. Here => [https://github.com/imirzadeh/Teacher-Assistant-Knowledge-Distillation/blob/master/train.py#L168](https://github.com/imirzadeh/Teacher-Assistant-Knowledge-Distillation/blob/master/train.py#L168
), define something like:   
`config = { "lambda_student": 0.5, "T_student": 5,  "seed": 20 }`   
rather than: 	`config = nni.get_next_parameter()`
